package.json

```diff
@@ -17,8 +17,11 @@
   },
```

Comment:
Add script for running worker node like `"worker-start": "node worker_node/index.js"`

Reply:
Done! Add `"start-worker": "node worker_nodex/index.js"` & Update readme